android/pytorch_android/src/androidTest/java/org/pytorch/PytorchInstrumentedTests.java

```diff
@@ -36,7 +36,7 @@ public void setUp() {
   public void testForwardNull() throws IOException {
     final Module module = Module.load(assetFilePath(TEST_MODULE_ASSET_NAME));
     final IValue input =
-        IValue.tensor(Tensor.newTensor(new long[] {1}, Tensor.allocateByteBuffer(1)));
+        IValue.tensor(Tensor.newByteTensor(new long[] {1}, Tensor.allocateByteBuffer(1)));
```

Comment:
maybe use uint8 to be consistent with PyTorch's API? (torch.uint8/int8)

Reply:
java byte is signed,so byteBuffer for tensor content can be filled with signed bytes.

To support uint8 we need some workaround, I think with method names to force that it is unsigned.

I think that uint8 and int8 naming wil be unnatural for java world...But maybe as we expect users be familiar with pytorch it should be ok. 

@dreiss , @ljk53 , what do you think about uint8, int8 naming in android java api ?